src/current/_includes/releases/v23.1/v23.1.26.md

```diff
@@ -0,0 +1,39 @@
+## v23.1.26
+
+Release Date: September 12, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+<h3 id="v23-1-26-security-updates">Security updates</h3>
+
+- For clusters using [Cluster SSO using JSON Web Tokens (JWTs)]({% link v23.1/sso-sql.md %}), the [cluster setting]({% link v23.1/cluster-settings.md %}) `server.jwt_authentication.issuers` can now take multiple values to support various kinds of providers and their mapped JWKS URI. You can set it to one of the following values, which are parsed by Go:
+  1. A string that contains a single valid issuer URI. Example: `https://accounts.google.com`
+  1. A string that contains a JSON array of valid issuer URIs. Example: `['example.com/adfs','https://accounts.google.com']`
+  1. A string that contains a JSON map of valid issuer URIs to corresponding JWKS URIs. and deserialized into a map of issuer URLs to corresponding JWKS URIs. A JSON map overrides the JWKS URI published in the issuer's `well-known/` endpoint. Example: `'{    "issuer_jwks_map": {        "https://accounts.google.com": "https://www.googleapis.com/oauth2/v3/certs",        "example.com/adfs": "https://example.com/adfs/discovery/keys"     }    }'`
+
+  When `issuer_jwks_map` is set, the key set is fetched from the JWKS URI directly. Otherwise, when `JWKSAutoFetchEnabled` is set, the JWKS URI is fetched from the issuer's `well-known/` endpoint. [#128669][#128669]
```

Comment:
```suggestion
    When `issuer_jwks_map` is set, the key set is fetched from the JWKS URI directly. Otherwise, when `JWKSAutoFetchEnabled` is set, the JWKS URI is fetched from the issuer's `well-known/` endpoint. [#128669][#128669]
```